de-lib/src/test/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/UpdatePermanentIdRequestDialogTest.java

```diff
@@ -59,17 +59,16 @@
     PermanentIdRequestDetails mockDetails;
 
     @Mock
-    UserBootstrap mockStrap;
+    UserProfile mockStrap;
```

Comment:
Since this is a `Mock` of a `UserProfile`, I would still call this `mockProfile`.